Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM ubuntu:16.04
+MAINTAINER Endless Services Team <services@endlessm.com>
+
+RUN apt-get update && \
+    apt-get install -y software-properties-common
+
+LABEL version="0.1"
+
+RUN add-apt-repository ppa:alexlarsson/flatpak
+
+RUN apt-get update && \
+    apt-get dist-upgrade -y && \
+    apt-get install -y apt-transport-https \
+                       curl \
+                       flatpak
+
+
+# Keep in line with requirements.txt and setup.py
```

Comment:
It's somewhat of a messy toss-up but generally you want the OS-provided libraries in Dockerfile as you can cache that layer easier and are usually more stable releases of code vs PIP. I understand the convenience that PIP provides but in this particular aspect, it's not ideal.